rv-iscroll.html

```diff
@@ -125,23 +134,56 @@
        */
       scrollY: {
         type: Boolean,
-        value: true
+        // value: true // ir: depreciating - boolean values are true whenever set so it's not possible to override with something like scroll-y="false"
+      },
+
+      /**
+       * Whether the component is ready
+       * @type {Boolean}
+       */
+      scrollY: {
+        type: Boolean,
+        // value: true // ir: depreciating - boolean values are true whenever set so it's not possible to override with something like scroll-y="false"
       },
 
     },
 
   ready: function () {
+	if(!this.scrollX && !this.scrollY)
+		this.scrollX = true;
```

Comment:
[iScroll docs](https://github.com/cubiq/iscroll#optionsscrollxoptionsscrolly) imply this should be `this.scrollY =  true`